### PR TITLE
KOGITO-3118 Infinispan 11 upgrade

### PR DESCRIPTION
**Pull request by cristianonicolai**
Assembly:
- https://github.com/kiegroup/kogito-runtimes/pull/791
- https://github.com/kiegroup/kogito-apps/pull/495
- https://github.com/kiegroup/kogito-examples/pull/407
- https://github.com/kiegroup/kie-docs/pull/2952

### CODE REVIEW

**Review by radtriste — APPROVED** [review]

File: onboarding-example/onboarding-quarkus/README.md [hunk 1]
```diff
@@ -32,20 +32,19 @@ Please note the additional parameter to specify you are running the service loca
 
 ### Running with persistence enabled
 
-Kogito supports runtime persistence that is backed by Inifinispan. So to be able to enable this you need to have 
-Infinispan server installed and available over the network. By default it expects it to be at (it can be configured via application.properties file located in src/main/resources)
+Kogito supports runtime persistence that is backed by Infinispan. So to be able to enable this you need to have 
+Infinispan server installed and available over the network. By default, it expects it to be at (it can be configured via application.properties file located in src/main/resources)
```
Comment (ricardozanini) [inline onboarding-example/onboarding-quarkus/README.md]: This sentence is a little bit odd to me. Do you mind rephrasing it?

File: kogito-travel-agency/extended/docker-compose/travel-agency-services-linux.yml [hunk 2]
```diff
@@ -3,7 +3,7 @@ version: '2.1'
 services:
 
   infinispan:
-    image: infinispan/server:10.0.0.Final-1
+    image: infinispan/server:11.0.4.Final
```
Comment (tiagodolphine) [inline kogito-travel-agency/extended/docker-compose/travel-agency-services-linux.yml:6]: I think it is necessary to configure the authentication information, users, and groups properties to make it work with ispn11, it is worth running the docker-compose from here just to confirm.

**Review by tiagodolphine — APPROVED** [review]
LGTM, just a minor comment.

**Review by MarianMacik — APPROVED** [review]

File: kogito-travel-agency/extended/docker-compose/travel-agency-services-linux.yml (hunk 2 above)
Comment (cristianonicolai) [reply kogito-travel-agency/extended/docker-compose/travel-agency-services-linux.yml:6]: @tiagodolphine thanks for the heads up, tested locally and seems to work just fine.

File: onboarding-example/onboarding-quarkus/README.md (hunk 1 above)
Comment (cristianonicolai) [reply onboarding-example/onboarding-quarkus/README.md]: @ricardozanini updated :)

**Comment by cristianonicolai** [comment]
jenkins please retest this

**Pull request merged**